README.md

```diff
@@ -25,6 +25,32 @@ You will need to clone the repository that contains the Vagrant/Ansible settings
 
     git clone git@github.com:datadryad/vagrant-dryad.git
     cd vagrant-dryad
+    
+## Preparing for an AWS deployment
+
+You need to make sure that you have credentials for AWS access and ssh access.
+
+You will need to have an access key ID and a secret access key for AWS. This is used to give your local machine permission to create VMs on the AWS infrastructure.
+
+Then, you'll need to create a keypair for yourself. This will be used by vagrant to log in to the VMs that you create, so only you will have access to these VMs unless you explicitly grant access to others. To create the keypair, log in to the aws.amazon.com console, then go to the EC2 dashboard. Click on "Key Pairs" on the left sidebar under "Network and Security." Then create a new key pair for yourself. The private key file `xxx.pem` should automatically download. Save this file somewhere safe on your machine, and note the path.
```

Comment:
"explicitly grant access to others" is not true now that we're automatically adding OpsWorks....